code/__HELPERS/text.dm

```diff
@@ -68,6 +68,26 @@
 /proc/adminscrub(t,limit=MAX_MESSAGE_LEN)
 	return copytext((html_encode(strip_html_simple(t))),1,limit)
 
+/**
+  * Perform a whitespace cleanup on the text, similar to what HTML renderers do
+  *
+  * This is useful if you want to better predict how text is going to look like when displaying it to a user
+  * HTML renderers collapse multiple whitespaces into one, trims prepending and appending spaces, among other things. This proc attempts to do the same thing.
+  * HTML5 defines whitespace pretty much exactly like regex defines the \s group, [ \t\r\n\f].
+  * Arguments:
+  * * t - The text to "render"
+  */
+/proc/htmlrendertext(t)
+	// Trim "whitespace" by lazily capturing word characters in the middle
+	var/regex/matchMiddle = new(@"^\s*([\W\w]*?)\s*$")
```

Comment:
make static as this will likely be called somewhat frequently, worth the cost of living in memory imo

```suggestion
	var/static/regex/matchMiddle = new(@"^\s*([\W\w]*?)\s*$")
```

Reply:
Thanks